main.go

```diff
@@ -47,7 +47,7 @@ func main() {
 	http.Handle("/metrics", promhttp.Handler())
 	go http.ListenAndServe(fmt.Sprintf(":%s", port), nil)
 
-	ac := controller.NewALBController(&aws.Config{MaxRetries: aws.Int(5)}, conf)
+	ac := controller.NewALBController(&aws.Config{MaxRetries: aws.Int(20)}, conf)
```

Comment:
yeah this makes sense, it will take a little more effort since the flags are parsed after this is created